content/ending-frozen-applications.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Closing Frozen Applications and Desktops
+description: >
+  Linux provides several methods to kill frozen or hung up processes and applications using GUI applications or terminal commands.
+keywords:
+  - frozen applications
+  - frozen desktop
+  - frozen terminal
+  - hung up
+  - not responding
+  - System76
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section:
+---
+
+## Closing Frozen Processes, Applications, and Desktops
+
+Linux offers many methods to stop hanging or frozen processes. The System Monitor application provides a GUI with features similar to the Windows Task Manager, while the `kill` command can end single or multiple processes from the Terminal.
+
+>Caution: Suddenly terminating processes can result in unwanted data loss or system stability issues.
+
+## Close Specific Applications or Windows
+
+### Using System Monitor (GUI)
+
+The System Monitor application comes prepackaged in Pop!\_OS. This tool provides a GUI and functionality similar to the Windows Task Manager.
+
+1. Press <kbd>Super</kbd> and type `system monitor`, then hit Enter.
+2. Locate frozen and hanging applications.
+   ![gnome-system-monitor](/images/ending-frozen-applications/gnome-system-monitor.png)
+3. Right click the application and click Stop in the dropdown menu.
```

Comment:
Sorry, this just hit me now that it's included in the screenshot-- why are we stopping (pausing) the frozen process instead of ending or killing it?